docs/tutorials/pytorch.ipynb

```diff
@@ -0,0 +1,507 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "45973fd5-6259-4e03-9501-02ee96f3f870",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Copyright (c) Microsoft Corporation. All rights reserved.\n",
+    "# Licensed under the MIT License."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9478ed9a",
+   "metadata": {
+    "id": "NdrXRgjU7Zih"
+   },
+   "source": [
+    "# Introduction to PyTorch\n",
+    "\n",
+    "_Written by: Adam J. Stewart_\n",
```

Comment:
Would like to start giving people credit for the tutorials they write, especially for the case study tutorials. This is intended to be the first person who wrote the tutorial, we wouldn't append other names except in the case of major revisions.

Reply:
great idea